workflow-examples/src/main/java/com/redhat/parodos/examples/continued/complex/ComplexWorkFlowConfiguration.java

```diff
@@ -66,7 +66,7 @@ WorkFlow assessmentWorkFlow(
 			@Qualifier("onboardingAssessmentTask") OnboardingAssessmentTask onboardingAssessmentTask) {
 		// @formatter:off
 		return SequentialFlow.Builder.aNewSequentialFlow()
-				.named("onboarding Assessment WorkFlow")
+				.named("onboardingAssessment" + WorkFlowConstants.ASSESSMENT_WORKFLOW)
```

Comment:
A question about this one: Isn't it redundant to specify the workflow type by annotation and also by the name?

Reply:
good question Moti! its name is just a metadata and it should be consistent from bean to builder(basically you can give a workflow any name). Annotation is which type it really is, we need to process a workflow based on its annotated type(e.g. assessment should return options,  checker should run on a schedule for certain tasks and so on).

Reply:
let me know if it makes sense

Reply:
@masayag Workflow type in parados is either Infrastructure, Assessment and Checker. Annotations haven't introduced to make that distinction including the need for additional metadata. On the other hand, workflow processing type (Sequential or Parallel) is derived form the workflow bean. Hope that helps clarifying the workflow type part.

Reply:
> Workflow type in parados is either Infrastructure, Assessment and Checker. Annotations

and Escalation since the weekend ;-) 

Thank you both for your replies. That makes sense to me.